src/App.js

```diff
@@ -1,10 +1,24 @@
 import MainPage from './MainPage'
 import styled from 'styled-components/macro'
+import ResultPage from './ResultPage'
+import useUserLocation from './hooks/useUserLocation'
```

Comment:
I like to cluster my imports a little. E.g. first styled, etc. then hooks, then components

Reply:
good point 👍🏻